src/stylesheets/components/_key-factors.scss

```diff
@@ -0,0 +1,291 @@
+/*----------- Key Factors -----------*/
+
+.key-factors {
+  color: $florence-gray;
+  margin-top: 150px;
+
+  @media ($screen-sm) {
+    margin-top: 65px;
+  }
+
+  h3 {
+    color: $florence-orange;
+    font-size: 1em;
+    margin-bottom: -8px;
+    text-transform: uppercase;
+  }
+}
+
+.section-title {
+  color: $florence-blue;
+  font-size: 0.65em;
+  letter-spacing: 2px;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.section-intro {
+  color: $florence-blue;
+  font-family: $secondary-font;
+  font-size: 1.7em;
+  line-height: 1.6em;
+  margin: auto;
+  text-align: center;
+  width: 57vw;
+
+  @media ($screen-sm) {
+    font-size: 1.4em;
+    width: 85vw;
+  }
+}
+
+.factor-bg {
+  background-image: url('../images/body-burst.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 1000px auto;
+  height: 740px;
+  margin-top: -110px;
+  width: 100%;
+
+  @media ($screen-sm) {
+    background-image: url('../images/body-burst-mobile.png');
+    background-position: top;
+    background-size: 100vw;
+    margin-top: -190px;
+    min-height: 1400px;
+  }
+
+  @media ($screen-xs) {
+    margin-top: -165px;
+  }
+}
+
+.factor-box {
+  background-color: $white;
+  border-radius: 9px;
+  font-size: 0.8em;
+  line-height: 1.4em;
+  overflow-x: visible;
+  padding: 25px;
+  position: relative;
+  width: 210px;
+
+  @media ($screen-sm) {
+    font-size: 0.9em;
+    left: 0;
+    line-height: 1.6em;
+    margin: 40px auto;
+    text-align: center;
+    top: 0;
+    width: 70vw;
+
+    & h3 {
+      letter-spacing: 1px;
+    }
+  }
+
+
+  &::before {
+    background-repeat: no-repeat;
+    background-size: 65px 65px;
+    bottom: 0;
+    content: '';
+    display: block;
+    height: 65px;
+    margin: auto;
+    position: absolute;
+    top: 20px;
+    width: 65px;
+  }
+
+  &-left {
+    border-radius: 156px 999px 999px 156px;
+    padding-right: 40px;
+    right: -56%;
+    top: 11%;
+
+    &:before {
+      right: -50px;
+      
+      @media ($screen-sm) {
+        background-size: 60px 60px;
+        left: -8vw;
+        top: -95%;
+      }
+
+      @media ($screen-xs) {
+        left: -13vw;
+      }
+    }
+
+    @media ($screen-sm) {
```

Comment:
define this before you define the `:before` pseudoclass so that it is easier to read